build/web/Dockerfile.multistage

```diff
@@ -0,0 +1,11 @@
+FROM cargo-infra.caicloud.xyz/library/node:9-alpine as build-deps
+WORKDIR /usr/src/app
+COPY web/package.json web/yarn.lock ./web/
```

Comment:
I'd prefer use WORKDIR directly, no need to add another `web`. Then also no need to `cd web`